profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single Profile
+**GET /api/cris/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the profile id that corresponds to the id of the eperson associated
```

Comment:
Is this ID the UUID of the EPerson? If so, we should state that more clearly.  Say something like `id: the UUID of the EPerson who owns the profile. This also corresponds to the ID of the profile.`